src/vscode-bicep/README.md

```diff
@@ -1,6 +1,6 @@
 # Key features of the Bicep VS Code extension
 
-The [bicep VS Code extension](./installing.md#install-the-bicep-vs-code-extension) is capable of many of the features you would expect out of other language tooling. Here is a comprehensive list of the features that are currently implemented.
+The [bicep VS Code extension](/docs/installing.md#install-the-bicep-vs-code-extension) is capable of many of the features you would expect out of other language tooling. Here is a comprehensive list of the features that are currently implemented.
```

Comment:
`vsce` packager prepends the repo base URL (`https://github.com/Azure/bicep/raw/main/`), hence why I'm making these into absolute URLs. This should be compatible with GitHub and the extension